data/strings/strings.txt

```diff
@@ -1571,49 +1571,48 @@
-    af = Voordat u die toep begin gebruik, moet u asb. die algemene wêreldkaart na u toestel aflaai.\nDit sal %@ se spasie opneem.
-    ar = قبل أن تبدأ باستخدام التطبيق، سنقوم بتنزيل خريطة العالم العامة على جهازك.\nستحتاج إلى %@ من البيانات.
-    az = Başlamazdan əvvəl gəlin ümumi dünya xəritəsini cihazınıza endirək.\n%@ yer tələb olunur.
-    be = Перад пачаткам працы дазвольце нам спампаваць агульную мапу свету на вашу прыладу.\nГэта патрабуе %@ сховішча.
-    bg = Преди да започнете да използвате приложението, позволете ни да изтеглим общата карта на света на вашето устройство.\nТова ще използва %@ от паметта.
-    ca = Abans d’usar l’aplicació, autoritzeu que baixem el mapa del món general al vostre aparell.\nUsarà %@ d’emmagatzematge.
-    cs = Ještě než začnete, bude třeba stáhnout obecnou mapu světa.\nZabere to %@.
-    da = Før du starter, så lad os downloade det generelle verdenskort til din enhed.\nIt behøver %@ af data.
-    de = Bevor Sie starten, laden Sie die allgemeine Weltkarte auf Ihr Gerät herunter.\nEs werden %@ Speicherplatz benötigt.
-    el = Πριν ξεκινήσετε τη χρήση της εφαρμογής, επιτρέψτε μας να κατεβάσουμε το γενικό παγκόσμιο χάρτη στη συσκευή σας.\nΘα χρησιμοποιήσει %@ αποθηκευτικού χώρου.
-    es = Antes de comenzar, permita que descarguemos en su dispositivo un mapamundi general.\nSe requieren %@ de almacenamiento.
-    et = Ennem rakenduse kasutamise alustamist luba meil laadida seadmesse alla maailma üldkaart.\nSee kasutab %@ mälumahtu.
-    eu = Hasi aurretik deskargatu mesedez mundu-mapa orokorra zure gailura.\nMemorian %@ erabiliko ditu.
+    en = Before you start using the app, please download the general world overview to your device.\nIt will use %@ of storage.
+    af = Voordat u die app begin gebruik, laai die General World -oorsig op u toestel af.\nDit sal %@ van die berging gebruik.
+    ar = قبل البدء في استخدام التطبيق، يرجى تنزيل النظرة العامة على العالم العام على جهازك.\nسيستخدم %@ من مساحة التخزين.
+    az = Tətbiqdən istifadə etməyə başlamazdan əvvəl, ümumi dünya icmalını cihazınıza yükləyin.\n%@ Anbar istifadə edəcəkdir.
+    be = Перш чым пачаць выкарыстоўваць прыкладанне, калі ласка, запампуйце агляд Агульнага свету на сваю прыладу.\nЁн будзе выкарыстоўваць %@ захоўвання.
+    bg = Преди да започнете да използвате приложението, изтеглете общия преглед на света на устройството си.\nТой ще използва %@ място за съхранение.
+    ca = Abans de començar a utilitzar l’aplicació, descarregueu la visió general del món general al vostre dispositiu.\nUtilitzarà %@ d’emmagatzematge.
+    cs = Než začnete aplikaci používat, stáhněte si do svého zařízení obecný přehled světa.\nVyužije %@ úložného prostoru.
+    da = Før du begynder at bruge appen, skal du downloade den generelle verdensoversigt til din enhed.\nDet vil bruge %@ af lagerpladsen.
+    de = Bevor Sie die App verwenden, laden Sie bitte die allgemeine Weltübersicht auf Ihr Gerät herunter.\nEs wird %@ des Speicherplatzes verwenden.
```

Comment:
```suggestion
    de = Bevor Sie die App verwenden, laden Sie bitte die weltweite Übersichtskarte herunter.\nEs werden %@ Speicherplatz benötigt.
```

(already includes "map" and excludes "general" as suggested in https://github.com/organicmaps/organicmaps/pull/10177#discussion_r1938383291)